### ISSUE TITLE
Measurement tool covers +/- zoom buttons

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/fourier-making-waves#204]
**Test device**
Jordan
**Operating System**
MacOS 10.13
**Browser**
Safari
**Problem description**
For https://github.com/phetsims/qa/issues/711.
The green lambda tool on the third screen will appear in such a way that the +/- buttons for the graph are fully visible, but can't be clicked easily since they are mostly covered in the area between the two end points. This isn't clear and can lead to the impression that the buttons aren't working. The measurement tool is easily moveable, but this is more of a concern if the tool appears like this.

**Visuals**
![scalescoverbuttons](https://user-images.githubusercontent.com/41024075/135899283-dcde4ffa-4413-46c5-9409-8a91f4952cda.gif)

**Comment by pixelzoom** [comment]
By design, measurement tools have pointer areas that are shaped like the solid red outline that I've Photoshopped into this screenshot:

<img width="1014" alt="screenshot_1306" src="https://user-images.githubusercontent.com/3046552/135902452-b6a6a0eb-a115-4805-bf3a-7045ec2bec48.png">

This makes it easy to drag the "calipers" part of the tool, and also makes the label draggable.  But it also means that things that are visible between the jaws of the calipers will not receive input events.  In the case that @KatieWoe has reported, that includes (but it not limited to) the zoom buttons.  _Anything_ that is behind this pointer area will not receive input events.

Also note that these pointer areas are not shown with `?showPointerAreas` because they are the default, and scenery does not display default pointer areas. 

@arouinfar would you like anything changed here?

**Comment by pixelzoom** [comment]
Raising priority to high, because QA is done with 1.0.0-rc.1.

**Comment by arouinfar** [comment]
It probably wouldn't be obvious to users that they are clicking on the caliper tool in @KatieWoe's gif, so the behavior looks buggy. However, I wouldn't change the pointer areas of the caliper tool. I also don't know that there should/could be more sophisticated detection in scenery to choose the UI component on the bottom in situations like these.

I don't have a good solution to offer here, other than to move the default position of the wavelength tool on the Wave Packet screen. Its default position is likely to cause overlap with the zoom buttons.
![image](https://user-images.githubusercontent.com/8419308/135937925-526048da-dedf-431c-9052-a8fae11d5813.png)

@pixelzoom let's move the default position up a bit
![image](https://user-images.githubusercontent.com/8419308/135937999-36f16497-699a-4683-8023-1d0247d2bbca.png)

**Comment by pixelzoom** [comment]
Good idea about moving the initial position of the tool.  The current initial position is a holdover from before we had checkboxes, someplease "out of the way".  So I like the position shown below even better, because it's definitely the best place to take measurements. So that's what I did (in master and 1.0).  @arouinfar please review. I can change it if you don't like it.

<img width="622" alt="screenshot_1311" src="https://user-images.githubusercontent.com/3046552/135939276-c87f86d4-8b8c-4d2b-a8b7-f9e53b72c5de.png">

**Comment by arouinfar** [comment]
Looks good to me, thanks @pixelzoom!

**Comment by pixelzoom** [comment]
Ready for testing in the next RC. To verify:

1. Run the sim
2. Go to Wave Packet screen
3. Check the checkbox for the lambda1 tool (green).
4. Verify that the tool's initial position is as shown in https://github.com/phetsims/fourier-making-waves/issues/204#issuecomment-933941459.
